sovrn-htb.js

```diff
@@ -152,7 +153,8 @@ function SovrnHtb(configs) {
                 contentType: false,
                 data: {
                     callback: 'window.' + SpaceCamp.NAMESPACE + '.' + __profile.namespace + '.adResponseCallback',
-                    br: JSON.stringify(br)
+                    br: JSON.stringify(br),
+                    src: `ix_${version}`
```

Comment:
Template literals can break in older browsers.  Use a string literal to maintain compatibility with older browsers.  Rather than creating a new version variable, the adapter version can be accessed using __profile.version.